bazel/envoy_build_system.bzl

```diff
@@ -1,7 +1,30 @@
-load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library", "py_proto_library")
-load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library")
 load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+load(":envoy_binary.bzl", _envoy_cc_binary = "envoy_cc_binary")
+load(":envoy_internal.bzl", "envoy_external_dep_path")
+load(
+    ":envoy_library.bzl",
+    _envoy_basic_cc_library = "envoy_basic_cc_library",
+    _envoy_cc_library = "envoy_cc_library",
+    _envoy_cc_posix_library = "envoy_cc_posix_library",
+    _envoy_cc_win32_library = "envoy_cc_win32_library",
+    _envoy_include_prefix = "envoy_include_prefix",
+    _envoy_proto_library = "envoy_proto_library",
+)
+load(
+    ":envoy_select.bzl",
+    _envoy_select_google_grpc = "envoy_select_google_grpc",
+    _envoy_select_hot_restart = "envoy_select_hot_restart",
+)
+load(
+    ":envoy_test.bzl",
+    _envoy_cc_fuzz_test = "envoy_cc_fuzz_test",
```

Comment:
So, is the issue that if you import `envoy_cc_fuzz_test` via `load`, it doesn't immediately give you a re-export?

Reply:
Yeah, exactly. Simply `load`ing it doesn't export the symbol. You have to declare it in the bzl file to export it. Even doing `load(foo = "foo")` doesn't export `foo`.